sites/partners/cypress/integration/application.spec.ts

```diff
@@ -9,15 +9,14 @@ describe("Application Management Tests", () => {
 
   it("Application grid should display correct number of results", () => {
     cy.visit("/")
-    cy.get(`[col-id="status"]`).eq(1).contains("Accepting Applications").click()
-    cy.getByID("lbTotalPages").contains("20")
+    cy.get(`[col-id="status"]`).eq(1).contains("Public").click()
     cy.get(".applications-table")
       .first()
       .find(".ag-center-cols-container")
       .first()
       .find(".ag-row")
       .should((elems) => {
-        expect(elems).to.have.length(8)
+        expect(elems).to.have.length(0)
```

Comment:
I worry a little that if the value here is zero, then the test is straying away from `Application grid should display correct number of results` - is there an easy way to have this table have applications in it so we can test fetching them?